src/plugins/custom_import_map/cypress/integration/geojson_file_upload.spec.js

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/// <reference types="cypress" />
+
+import { BASE_PATH } from '../utils/constants';
+import 'cypress-file-upload';
+
+describe('Verify successful custom geojson file upload', () => {
+  before(() => {
+    cy.visit(`${BASE_PATH}/app/home#/tutorial_directory/sampleData`);
+    // Click on "Sample data" tab
+    cy.contains('Sample data').click({ force: true });
```

Comment:
could you try adding a timeout in this line? something like `cy.contains('Sample data, { timeout: 60000 }).click({ force: true });`

Reply:
This worked like magic!